### PR TITLE
Migration code cleanup

### PR DESCRIPTION
**Pull request by SINIKI**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Core/PolicySetup/PolicySetup.ps1 [hunk 1]
```diff
@@ -439,16 +398,8 @@ class PolicySetup: CommandBase
...
-		{
-			& $this.MigrationScriptPath
-		}
-		else
-		{
-			$mgrationScript = $this.LoadServerConfigFile("PolicyMigration.ps1")			
-			Invoke-Expression $mgrationScript
-		}
+		$mgrationScript = $this.LoadServerConfigFile("PolicyMigration.ps1")			
+		Invoke-Expression $mgrationScript
```
Comment (sbyna-msft) [inline src/AzSK/Framework/Core/PolicySetup/PolicySetup.ps1]: @SINIKI  why these things are still required?

Comment (SINIKI) [reply src/AzSK/Framework/Core/PolicySetup/PolicySetup.ps1]: As discussed with @VishalHaibatpure, he will be removing Policy Migration related code in a separate PR.

File: src/AzSK/Framework/Core/PolicySetup/PolicySetup.ps1 [hunk 2]
```diff
@@ -228,34 +217,6 @@ class PolicySetup: CommandBase
...
 			$azskOverride.WriteToFolder($this.ConfigFolderPath);
 		}
-		elseif($this.IsMigrationOn)
-		{
-			$azskOverride = [ConfigOverride]::new($this.ConfigFolderPath,"AzSK.json");
-			if([Helpers]::CheckMember($azskOverride.ParsedFile, "PolicyMessage"))
-			{
-				$PolicyMessage = $azskOverride.ParsedFile.PolicyMessage
-				$PolicyMessage = $PolicyMessage.Replace($([Constants]::OldModuleName),$moduleName)
-				$azskOverride.UpdatePropertyValue("PolicyMessage", $PolicyMessage);
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/PolicySetup/PolicySetup.ps1]: Need to validate and remove the constants defined for migration things like [Constants]::OldModuleName

Comment (SINIKI) [reply src/AzSK/Framework/Core/PolicySetup/PolicySetup.ps1]: As discussed with Byna, we will not delete this variable as user can still download AzSDK from PS Gallery.

**Pull request merged**